docs/configuration.asciidoc

```diff
@@ -273,6 +273,9 @@ This option supports the wildcard `*`, which matches zero or more characters.
 Examples: `/foo/*/bar/*/baz*`, `*foo*`. Matching is case insensitive by default.
 Prefixing a pattern with `(?-i)` makes the matching case sensitive.
 
+NOTE: `ELASTIC_APM_IGNORE_URLS` is deprecated, and will be removed in a future major
```

Comment:
```suggestion
NOTE: this configuration was previously known as `ELASTIC_APM_IGNORE_URLS`, which has been deprecated and will be removed in a future major
```

Otherwise people unfamiliar with existing config might think it's a typo, or even glance over the difference, and think it refers to `ELASTIC_APM_TRANSACTION_IGNORE_URLS`